be/src/exec/parquet/column_reader.cpp

```diff
@@ -35,6 +35,127 @@ class ColumnConverter {
     virtual Status convert(const vectorized::ColumnPtr& src, vectorized::Column* dst) = 0;
 };
 
+class Int64ToDateTimeConverter : public ColumnConverter {
```

Comment:
nit: how about adding an abstraction for Int96ToDateTimeConverter and Int64ToDateTimeConverter?